.github/workflows/lint-md.yml

```diff
@@ -0,0 +1,11 @@
+name: Check Markdown links
+on:
+  pull_request:
+    paths: "**.md"
+jobs:
+  lint-md:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: npm install
+      - run: npx remark --use validate-links --frail . .github
```

Comment:
I'm a little nervous about this hidden dependency.  It seems like it would be harder to overlook if it were declared in package.json.

Having said that, I think @orta already has some code for detecting dead links and I'm not sure this is consistent with the approach we use elsewhere.

Reply:
> I'm a little nervous about this hidden dependency. It seems like it would be harder to overlook if it were declared in package.json.

This change does declare those dependencies in `package.json`, if I understand your comment correctly? https://github.com/DefinitelyTyped/DefinitelyTyped/blob/fe72bd3fdc8af7c6482de1819497913ba0ee1a04/package.json#L33-L34

> Having said that, I think @orta already has some code for detecting dead links and I'm not sure this is consistent with the approach we use elsewhere.

I'm all in favor of an existing solution :+1:

Reply:
I have an offline one for the TS website, but I think this is a good strategy